### DISCUSSION TITLE
Welcome to purple-octopus Discussions!

### DISCUSSION THREAD

**Discussion by sillyfalcon** [sillyfalcon/purple-octopus#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shaswathpatil** [comment]
I am able to see only raw code while reviewing. 
Is there a way to see which python code lines were added when commit is made ?
